### ISSUE TITLE
Cannot compile on Arch without adding '-ltinfo' to Makefile

### ISSUE THREAD

**Issue by sloan-848** [dpayne/cli-visualizer#71]
When installing with the provided instructions in the README, the project was failing to build with the following error:
```bash
/usr/bin/ld: /tmp/cli-visualizer-1.6/build/Visualizer.o: undefined reference to symbol 'wtimeout'
/usr/lib/libtinfo.so.6: error adding symbols: DSO missing from command line
```

I resolved this issue by adding `-ltinfo` to the `LIBS` variable in the Makefile. 

This library may automatically be automatically included on other systems, but it wouldn't seem to hurt to explicitly include it, since `ncurses` is already a dependency for building.

**Comment by sloan-848** [comment]
Addressed in #72

**Comment by dpayne** [comment]
Fix has been merged.
